### PR TITLE
Add default light and toggle button

### PR DESCRIPTION
**Pull request by wherget**
This Branch should add a Light shining in the direction of the camera, as well as a button to toggle that light on and off.

### CODE REVIEW

File: compass-js-frontend/src/main/webapp/ui/templates/editor/navbar/defaultlight.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="btn-group btn-group-default">
+	<button type="button"
+		id="defaultLightButton"
```
Comment (wherget) [inline compass-js-frontend/src/main/webapp/ui/templates/editor/navbar/defaultlight.html]: Does this even need an ID anymore? I don't think it's referenced anywhere.

**Comment by wherget** [comment]
I'll give it a shot with the merge, see what the conflicts are.

File: compass-js-frontend/src/main/webapp/ui/templates/editor/viewport.html [hunk 2]
```diff
@@ -2,12 +2,11 @@
 	<xml3d activeview="#defaultView" class="xml3d">
 		<defs id="mainDefs">
 		</defs>
-		<view id="defaultView" position="1 1.5 13"></view>
+		<view id="defaultView" position="1 1.5 13">
+			<group data-hook="defaultLight"></group>
```
Comment (ebersold) [inline compass-js-frontend/src/main/webapp/ui/templates/editor/viewport.html]: A group within the view node, is this even supported by XML3D?

Comment (wherget) [inline compass-js-frontend/src/main/webapp/ui/templates/editor/viewport.html]: That's supposed to be the placeholder for the the light. When I wrote it my thought process was that `<group>` is basically XML3D's neutral element (much like `<div>` in html).

Comment (wherget) [inline compass-js-frontend/src/main/webapp/ui/templates/editor/viewport.html]: To be fair though, I don't think anything is supported below a `<view>`, at least officially. Need to check whether it (by random chance) works while merging.

File: compass-js-frontend/src/main/webapp/ui/templates/editor/xml3d/default-light.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<light id="defaultLight" model="urn:xml3d:light:point">
```
Comment (wherget) [inline compass-js-frontend/src/main/webapp/ui/templates/editor/xml3d/default-light.html]: Wasn't the JSF default light a directional light?

Comment (ebersold) [inline compass-js-frontend/src/main/webapp/ui/templates/editor/xml3d/default-light.html]: The old JSF default light was a point light as well.

**Comment by wherget** [comment]
I'm fine with this now. :+1: :shipit:

**Pull request merged**